instrumentation-core/spring/spring-web-3.1/spring-web-3.1.gradle

```
@@ -3,6 +3,7 @@ ext{
 }
 
 apply from: "$rootDir/gradle/instrumentation-library.gradle"
+archivesBaseName = "spring-web"
```

it's a little weird what we do here, but it helps when we need to support multiple versions of a library (e.g. see lettuce)
```suggestion
archivesBaseName = "spring-web-3.1"
```